src/tally.h

```diff
@@ -86,13 +104,17 @@ namespace pyne
     void write_hdf5(std::string filename, std::string datapath);
 
     // mcnp tally
-    std::string mcnp(int tally_index = 1, std::string mcnp_version = "mcnp5" );
-  
+    std::string mcnp(int tally_index = 1, std::string mcnp_version = "mcnp5",
```

Comment:
why isn't `out` part of the tally class member variables?

Reply:
I was thinking user would provide it when writing the tally... so it provide more freedom
maybe it is'nt a good idea....

Reply:
I guess I see that use case... I'm kind of on the fence....  The user could also just change the class member variable before writing the it

Reply:
I guess it depends on the canonical definition of a mesh tally, if it includes the "OUT" variable, then yes it should be there, otherwise I guess it should be an argument of the `mcnp()` method....

Does it make sense ?